test/rubocop/cop/minitest/assert_equal_arguments_order_test.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class AssertEqualArgumentsOrderTest < Minitest::Test
+  def test_registers_offense_when_actual_is_basic_literal
+    assert_offense(<<~RUBY)
+      class FooTest < Minitest::Test
+        def test_do_something
+          assert_equal(foo, 2)
+          ^^^^^^^^^^^^^^^^^^^^ You misordered expected and actual arguments for `assert_equal`.
```

Comment:
I think the method name is fine. E.g.:

```suggestion
          assert_equal(foo, 2)
                       ^^^^^^ You misordered expected and actual arguments for `assert_equal`.
```